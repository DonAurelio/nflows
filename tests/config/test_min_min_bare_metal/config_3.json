{
    "dag_file": "./tests/workflows/test_min_min_bare_metal/config_3.dot",

    "scheduler_type": "min-min",
    "scheduler_params": [],

    "mapper_type": "bare-metal",
    "mapper_mem_policy_type": "default",
    "mapper_mem_bind_numa_node_ids": [],

    "core_avail_mask": "0x1000001",
    "flops_per_cycle": 32,
    "clock_frequency_type": "static",
    "clock_frequency_hz": 1000000000,

    "distance_matrices": {
        "latency_ns": "./tests/system/test_min_min_bare_metal/3_lat.txt",
        "bandwidth_gbps": "./tests/system/test_min_min_bare_metal/3_bw.txt"
    },

    "out_file_name": "./tests/output/test_min_min_bare_metal/config_3.yaml"
}
