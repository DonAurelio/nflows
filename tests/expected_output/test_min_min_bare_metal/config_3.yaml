core_availability:
  0: {avail_until: 0}
  1: {avail_until: 0}

name_to_thread_locality:
  Task_3: {numa_id: 0, core_id: 0}
  Task_2: {numa_id: 0, core_id: 1}
  Task_1: {numa_id: 0, core_id: 0}
